data/coalition/coalition missions.txt

```diff
@@ -3111,3 +3111,33 @@ mission "Coalition: Outskirts Gauger Ready"
 			`	While the Arachi work on the outfit, she explains that they used the probe's algorithms to have the scanners excel at finding basic energy signatures, such as heat or energy levels of other ships. "Made to check for complex machinery like outfits, or recognize specific cargo types, it was not, but still find uses for its current form, I'm sure you will."`
 			`	When everything's ready, you two head back inside your ship, and the elevator brings you back up. You fly back to the main landing area, and she pulls close to you just before you open your hatch. "Now, still... experimental, the outfit is, and provide the Heliarchs with a potentially faulty product, we do not wish. So, naturally, not yet registered it with them, we have. Provide you with more copies, we cannot, so careful not to lose this one, you should be." She thanks you again for helping out, and leaves for her workplace.`
 				decline
+
+
+
+mission "Heliarchs Buy Jump Drive Mission"
+	minor
+	landing
+	invisible
+	to offer
+		has "Coalition: First Contact: done"
+		has "outfit (cargo): Jump Drive"
```

Comment:
Something to note: this will offer the first time a person comes here (in the first contact mission) if they happen to have a jump drive in cargo.
Perhaps that conversation should be changed if the player has an extra jump drive when they do it.

Reply:
If that's acknowledged in the First Contact conversation it'd be awkward not getting a prompt then to sell it, but that feels it'd be intruding into a convo that's not supposed to have anything to do with getting them more JDs.